From 044dbdeec1f7216e78fdba0a9dcfb3bfd13e3000 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Mon, 23 Jan 2012 15:21:26 -0800
Subject: [PATCH 192/236] MIPS: Octeon: Use CIU2_INTR_CIU_READY for all cn68XX.

Source: Cavium SDK 2.3-427

CN68XX pass2 seems to need this too.

Signed-off-by: David Daney <david.daney@cavium.com>
Integrated-by: Jiang Bin <bin.jiang@windriver.com>
---
 arch/mips/cavium-octeon/octeon-irq.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/cavium-octeon/octeon-irq.c b/arch/mips/cavium-octeon/octeon-irq.c
index a54696c..36ed0ff 100644
--- a/arch/mips/cavium-octeon/octeon-irq.c
+++ b/arch/mips/cavium-octeon/octeon-irq.c
@@ -1401,7 +1401,7 @@ spurious:
 out:
 	/* CN68XX pass 1.x has an errata that accessing the ACK registers
 		can stop interrupts from propagating */
-	if (OCTEON_IS_MODEL(OCTEON_CN68XX_PASS1_X))
+	if (OCTEON_IS_MODEL(OCTEON_CN68XX))
 		cvmx_read_csr(CVMX_CIU2_INTR_CIU_READY);
 	else
 		cvmx_read_csr(CVMX_CIU2_ACK_PPX_IP2(core_id));
@@ -1428,7 +1428,7 @@ spurious:
 out:
 	/* CN68XX pass 1.x has an errata that accessing the ACK registers
 		can stop interrupts from propagating */
-	if (OCTEON_IS_MODEL(OCTEON_CN68XX_PASS1_X))
+	if (OCTEON_IS_MODEL(OCTEON_CN68XX))
 		cvmx_read_csr(CVMX_CIU2_INTR_CIU_READY);
 	else
 		cvmx_read_csr(CVMX_CIU2_ACK_PPX_IP3(core_id));
-- 
1.7.0

